Rules/Strings.resx

```diff
@@ -1104,4 +1104,16 @@
   <data name="UseProcessBlockForPipelineCommandName" xml:space="preserve">
     <value>UseProcessBlockForPipelineCommand</value>
   </data>
+  <data name="ReviewUnusedParameterCommonName" xml:space="preserve">
+    <value>ReviewUnusedParameter</value>
+  </data>
+  <data name="ReviewUnusedParameterDescription" xml:space="preserve">
+    <value>Ensure all parameters are used within the same script, scriptblock, or function where they are declared.</value>
+  </data>
+  <data name="ReviewUnusedParameterError" xml:space="preserve">
+    <value>The parameter {0} has been declared but not used. </value>
```

Comment:
```suggestion
    <value>The parameter '{0}' has been declared but not used. </value>
```

Reply:
fixed in 938a2835ac8d3c8e5470642d923c3b85476b303c